plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/CountReportingPolicy.java

```diff
@@ -19,10 +19,14 @@
 import static java.util.Arrays.asList;
 
 /**
+ * Synthesizes a count from remote and hosted component counts.
+ *
  * @since 3.0
  */
 public class CountReportingPolicy
 {
+  private CountReportingPolicy() {}
```

Comment:
In maven plugin this raised sonar "codestyle" warning, and required to put closing bracket on next line...